<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckAvailableRoom</class>
 <widget class="QMainWindow" name="CheckAvailableRoom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="Reservable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>411</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>0</y>
      <width>231</width>
      <height>152</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Available Single Room  </string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLineEdit" name="lineEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Available Double Room</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Available Royal Suite  </string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Available Cars</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Available Busses</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="4">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Available Limousine Car</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QLineEdit" name="lineEdit_6"/>
     </item>
     <item row="1" column="4">
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
     <item row="2" column="4">
      <widget class="QLineEdit" name="lineEdit_3"/>
     </item>
     <item row="3" column="4">
      <widget class="QLineEdit" name="lineEdit_4"/>
     </item>
     <item row="4" column="4">
      <widget class="QLineEdit" name="lineEdit_5"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>401</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="RoomStatus">
       <property name="text">
        <string>Room Status</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="VehicleStatus">
       <property name="text">
        <string>Vehicle Status</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>673</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
